nextflow.config

```diff
@@ -66,6 +66,10 @@ params {
     min_mapped_reads           = 5
     extra_star_align_args      = null
     extra_salmon_quant_args    = null
+    extra_kallisto_quant_args  = null
+    kallisto_quant_fraglen     = 200
+    kallisto_quant_fraglen_sd  = 200
+    kallisto_quant_strandedness  = null
```

Comment:
OK, this got lost when I rearranged the module config after your feedback- I'll try to work it back in

Reply:
I don't actually think we need it as a standalone parameter, it can go in via the kallisto extra args. We just have to make sure the module sees it and doesn't override it, which I've done. 

https://github.com/nf-core/rnaseq/pull/1106/commits/6c75322180c7743a68c6815fa46f91f4424721b8